### ISSUE TITLE
Edit/Finish styling for Meta Fields

### ISSUE THREAD

**Issue by patrickdemooij9** [patrickdemooij9/uSeoToolkit.Umbraco#27]
Two changes:

- See if we can move the edit button inside of the group header. In this case, we can get rid of some of the whitespacing there.
- Instead of the finish button being where the edit button is. We should have the publish bar appear and show a save button. Will make it feel more like Umbraco this way.

**Comment by leekelleher** [comment]
> Instead of the finish button being where the edit button is. We should have the publish bar appear and show a save button. Will make it feel more like Umbraco this way.

This is the JS that I'd mentioned in my review. It'll make sure the Save/Published buttons are displayed on the ContentApp.

```js
(function () {
    "use strict";

    function SeoSettingsDecorators($provide) {

        $provide.decorator("contentAppHelper", ["$delegate", function contentAppHelperDecorator($delegate) {

            if ($delegate.CONTENT_BASED_APPS.indexOf("metaFieldsSeoSettings") === -1) {
                $delegate.CONTENT_BASED_APPS.push("metaFieldsSeoSettings");
            }

            return $delegate;
        }]);
    };

    angular.module("umbraco.services").config(["$provide", SeoSettingsDecorators]);
}());
```

But with this, I'm unsure where the Edit/Finished workflow logic fits into it?
Would the Meta Fields already be in an editable state? 🤔
I'm not sure I've helped reach a solution here. 😆

**Comment by patrickdemooij9** [comment]
Yeah, this might be a difficult one to make clear 🤔 Or at least make it feel Umbraco like.
Starting with an editable state doesn't really fix it, because there still needs to be a way to show the user what the value is if they leave it in an empty state. And (judging from the clients we have) the edit functionality probably won't get used that often.

Will see if I come up with something this afternoon
